os_update-formula/pillar.example

```diff
@@ -0,0 +1,18 @@
+os-update:
+  # by default, the timer is left disabled
+  # to enable it, set this to true
+  enable: false
+
+  # by default, no timer override will be installed, and the packaged default will apply
+  # to install an override, use the syntax described in systemd.timer(5) - example:
+  # timer: 'Thu *-*-01/4 02:00:00'
+  timer: false
```

Comment:
`timer: false` might be misunderstood as "do not enable os-update.timer".

Maybe use `time:` instead?

Reply:
Good idea.

Reply:
Changed.